### PR TITLE
NMS-13695: Migrate Discourse KSC Reports article to main docs

### PR DESCRIPTION
**Pull request by mfuhrmann**
### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13695

### CODE REVIEW

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
a few changes.

File: docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc [hunk 1]
```diff
@@ -12,5 +14,38 @@
...
+  "Yesterday 5pm-10pm", "This Week", "Last Week", "This Month",
+  "Last Month", "This Quarter", "Last Quarter", "This Year",
+  "Last Year"
+----
+
+Be sure to keep each `id` unique for each `<Report>`.`
+
+## Using KSC Reports
+
+On the right of the main page of OpenNMS there is a drop-down selection box with the title `KSC Reports` where you can select the KSC reports you have configured.
```
[the remark this answers is not in the document]
Comment (mmahacek) [reply docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc]: Correct, it's a search box.

[the remark this answers is not in the document]
Comment (mfuhrmann) [reply docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc]: @Bonrob2 Tested in version 29. It's a list and a search bar. No drop-down for me.

image

[the remark this answers is not in the document]
Comment (mfuhrmann) [reply docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc]: @Bonrob2 I've edited you suggestion here https://github.com/OpenNMS/opennms/pull/4087#discussion_r781634176
If ok please commit it.

[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc]: @mfuhrmann There are some conflicting files that we need to resolve.

**Review by Bonrob2 — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]

File: docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc [hunk 2]
```diff
@@ -12,5 +15,38 @@
...
+  "Yesterday 5pm-10pm", "This Week", "Last Week", "This Month",
+  "Last Month", "This Quarter", "Last Quarter", "This Year",
+  "Last Year"
+----
+
+Be sure to keep each `id` unique for each `<Report>`.`
+
+== Using KSC Reports
+
+The *KSC Reports" area on the right side of the main page of OpenNMS includes a drop-down menu where you can select the KSC reports you have configured.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/admin/webui/opsboard/dashlet/ksc.adoc]:
```suggestion
The *KSC Reports* area on the right side of the main page of OpenNMS includes a drop-down menu where you can select the KSC reports you have configured.
```
My bad. I made a typo.

**Review by Bonrob2 — DISMISSED** [review]

**Review by Bonrob2 — APPROVED** [review]

**Pull request merged**